rewriter/Minitest.cc

```diff
@@ -145,6 +143,90 @@ string to_s(core::Context ctx, unique_ptr<ast::Expression> &arg) {
     return arg->toString(ctx);
 }
 
+// This returns `true` for expressions which can be moved from class to method scope without changing their meaning, and
+// `false` otherwise. This mostly encompasses literals (arrays, hashes, basic literals), constants, and sends that only
+// involve the other things described.
```

Comment:
What's a basic literal?

Reply:
a constant literal?

Reply:
By "basic literal" I meant literals that are, uh, `ast::Literal` (i.e. numbers, strings, &c).